app/layout/nav_menu.phtml

```diff
@@ -77,8 +78,12 @@
 				'idMax' => FreshRSS_Context::$id_max,
 				'search' => htmlspecialchars_decode(FreshRSS_Context::$search, ENT_QUOTES),
 				'state' => FreshRSS_Context::$state,
-			)
+			),
```

Comment:
I always get annoyed by languages that don't annoy this. ;-)

Reply:
Like JSON!